src/Http/Controllers/ProfileInformationController.php

```diff
@@ -4,8 +4,12 @@
 
 use Illuminate\Http\Request;
 use Illuminate\Routing\Controller;
+use Illuminate\Support\Str;
 use Laravel\Fortify\Contracts\ProfileInformationUpdatedResponse;
 use Laravel\Fortify\Contracts\UpdatesUserProfileInformation;
+use Laravel\Fortify\Fortify;
+
+use function config;
```

Comment:
Can you also remove all `use function config` and `use function app` imports? They're not needed. Thanks